graphql/database/experiments/features.js

```diff
@@ -135,6 +136,66 @@ const features = {
       },
     ],
   },
+  [YAHOO_SEARCH_NEW_USERS_V2]: {
+    defaultValue: 'Tooltip',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Control',
+            $exists: true,
+          },
+        },
+        force: 'Control',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Tooltip',
+            $exists: true,
+          },
+        },
+        force: 'Tooltip',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Notification',
+            $exists: true,
+          },
+        },
+        force: 'Notification',
```

Comment:
The notification behavior needs updating, such as in [getShouldShowYahooPrompt.js](https://github.com/gladly-team/tab/blob/92c8f855d0d5e2f6369a12bc8b7e41834e091e11/graphql/database/users/getShouldShowYahooPrompt.js), before it's functional.